src/applications/disability-benefits/all-claims/pages/vaMedicalRecords.js

```diff
@@ -93,9 +93,8 @@ export const schema = {
     },
     vaTreatmentFacilities: merge({}, vaTreatmentFacilities, {
       items: {
-        required: ['treatmentCenterName', 'relatedDisabilities'],
```

Comment:
?

Reply:
This was only in here before to add `relatedDisabilities` to the required list because it wasn't in the schema, but now it _is_ in the schema--and required to boot!

Seemed silly to override the property with the same values is all.

Reply:
Well that sure makes sense